docs/functions/vRealize Suite/New-VCFvRSLCM.md

```diff
@@ -0,0 +1,58 @@
+# New-VCFvRSLCM
+
+### Synopsis
+Deploys vRealize Suite Lifecycle Manager
+
+### Syntax
+```
+New-VCFvRSLCM -json <path to json file>
+```
+
+### Description
+Deploys vRealize Suite Lifecycle Manager based on a JSON spec. Supports vLan or AVN based deployments.
```

Comment:
Use 'The New-VCFvRSLCM cmdlet deploys...'
Change 'vLan' to VLAN'
Change 'AVN' to 'Application Virtual Network' to ensure we are spelling it out (folks may not know what AVN stands for)